mixnet/topology/src/lib.rs

```diff
@@ -9,17 +9,17 @@ pub type MixnetNodeId = [u8; PUBLIC_KEY_SIZE];
 
 pub type Result<T> = core::result::Result<T, NymNodeRoutingAddressError>;
 
-#[derive(Serialize, Deserialize, Clone, Debug, Default)]
+#[derive(Serialize, Deserialize, Clone, Debug, Default, Eq, PartialEq)]
```

Comment:
why this?

Reply:
I added this for the test that is added in this PR, in order to compare MixnetClientConfigs objects

Reply:
Ah, actually, we need only PartialEq (not Eq) for the test.